Mathlib/AlgebraicGeometry/ResidueField.lean

```diff
@@ -0,0 +1,119 @@
+/-
+Copyright (c) 2024 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.AlgebraicGeometry.OpenImmersion
```

Comment:
Why is this necessary? I think everything `LocallyRingedSpace`-related should be in `Geometry/RingedSpace/OpenImmersion`.